lib/stm32/common/exti_common_all.c

```diff
@@ -41,8 +41,35 @@
 	#define EXTICR_SELECTION_REG(x)	SYSCFG_EXTICR(x)
 #endif
 
+
+/*---------------------------------------------------------------------------*/
+/** @brief Set trigger  [31:0] extended interrupts.
+
+Set trigger values   extended [31:0] interrupts
+
+@param[in] extis extended interrupt number [31:0]  @ref exti_values.
+@param[in] trig extended interrupt front type @ref exti_trigger_values.
+*/
+
 void exti_set_trigger(uint32_t extis, enum exti_trigger_type trig)
 {
+
+#if defined(EXTI_RTSR1) && defined(EXTI_FTSR1)
+	switch (trig) {
+	case EXTI_TRIGGER_RISING:
+		EXTI_RTSR1 |= extis;
+		EXTI_FTSR1 &= ~extis;
+		break;
+	case EXTI_TRIGGER_FALLING:
+		EXTI_RTSR1 &= ~extis;
+		EXTI_FTSR1 |= extis;
```

Comment:
no no no. we normally provide an alias for xxxx1 to xxxx for things like this in past, ADC1 and ADC, and DAC1 and DAC for instance. you can use the same thing here.